tests/components/homekit/test_type_lights.py

```diff
@@ -13,6 +13,10 @@
 
 from tests.common import get_test_home_assistant
 
+patch('homeassistant.components.homekit.accessories.debounce',
```

Comment:
This is bad as now loading this file impacts all other tests that use homekit.

Instead, you should use a pytest fixture. But to be able to leverage pytest fixtures, you will need to rewrite your tests to be just methods. Something like this: (written from top of my head, didn't test)

```python
import pytest

@pytest.fixture(autouse=True)
def mock_debounce():
    with patch('…'):
        yield


@pytest.fixture
def mock_call_service(hass):
    events = []

    @callback
    def record_event(event):
        …

    hass.bus.listen(…)
    return events


def test_light_basic(hass, mock_call_service):
    # hass is a new test instance, will be automatically stopped for you
    # mock_call_service is equivalent to self.events
    # Instead of self.assertEqual, just use assert
    assert acc.aid == 2
```